index.d.ts

```diff
@@ -1434,10 +1434,19 @@ export interface FileUploaderOwnProps extends FormFieldOwnProps {
    * @see https://developer.mozilla.org/en-US/docs/Web/HTTP/Basics_of_HTTP/MIME_types/Common_types
    */
   acceptedMimeTypes?: MimeType[]
+  /**
+   * Function to return a string or component for the 'Browse or drag' text
+   */
+  browseOrDragText?: (maxFiles: number) => React.ReactNode
```

Comment:
What is the difference between this call and the `dragMaxFilesMessage`? In this call we pass in the `maxFiles` and then return a string or component vs the other it only returns a `string` when the `maxFiles` size has been hit on drag.

Reply:
`browseOrDragText` is the text inside the dropzone, while `dragMaxFilesMessage` is the validation message that appears underneath the dropzone on drag:

![image](https://user-images.githubusercontent.com/11774799/197838928-27548e33-d300-4dce-966c-720d5694acc9.png)